assets/js/hooks/copy_hook.js

```diff
@@ -1,12 +1,36 @@
 export default {
   mounted() {
+    const copySvg = `
```

Comment:
Let's not mix HTML and JS code. Put these two icons in the HEEX template and then toggle them (show one and hide the other one) once the command has been copied, and then revert after the timeout.